doc/DeepSpeech.md

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
Sphinx looks interesting. I kept @tilmankamp's approach here without really questioning it. Maybe other options are not as easy to publish on GitHub?